sale_stock_picking_note/models/stock_picking.py

```diff
@@ -1,10 +1,38 @@
 # Copyright 2021 Tecnativa - Víctor Martínez
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
 
-from odoo import fields, models
+from odoo import api, fields, models
 
 
 class StockPicking(models.Model):
     _inherit = "stock.picking"
 
-    customer_note = fields.Text(string="Customer Comments")
+    customer_note = fields.Text(
+        string="Customer Comments",
+        compute="_compute_picking_notes",
+        store=True,
+        readonly=False,
+    )
+    note = fields.Html(
+        compute="_compute_picking_notes",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends(
+        "sale_id.picking_note",
+        "partner_id.picking_note",
+        "sale_id.picking_customer_note",
+        "partner_id.picking_customer_note",
+        "picking_type_id",
+    )
+    def _compute_picking_notes(self):
```

Comment:
For the same reasons as explained above, I also think a compute is not the best solution here.